ros/src/models/chrono/ros_chrono/src/steering_controller.cpp

```diff
@@ -402,6 +405,14 @@ int main(int argc, char* argv[]) {
     double throttle_input = 0;			///
     double steering_input = 0;			///
     double braking_input = 0;			///
+    
+    bool is_init;
+    node.getParam("system/nloptcontrol_planner/flags/initialized", is_init);
```

Comment:
you are not waiting for my planner. You are waiting for the sytem
https://github.com/JuliaMPC/MAVs/blob/develop/ros/src/system/bootstrap.jl#L75

change to
``` 
   node.getParam("system/flags/initialized", is_init);
```